tests/UnitTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace RonasIT\Media\Tests;
+
+use Illuminate\Http\Testing\File;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
+use RonasIT\Media\Contracts\Services\MediaServiceContract;
+use RonasIT\Media\Enums\PreviewDriverEnum;
+use RonasIT\Media\MediaRouter;
+use RonasIT\Media\Models\Media;
+use RonasIT\Media\Tests\Support\MediaTestTrait;
+use RonasIT\Media\Tests\Support\ModelTestState;
+use RonasIT\Support\Traits\FilesUploadTrait;
+
+class UnitTest extends TestCase
+{
+    use FilesUploadTrait;
+    use MediaTestTrait;
+
+    protected static File $file;
+    protected static ModelTestState $mediaTestState;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        self::$file ??= UploadedFile::fake()->image('file.png', 600, 600);
+        self::$mediaTestState ??= new ModelTestState(Media::class);
+
+        Storage::fake();
+
+        MediaRouter::$isBlockedBaseRoutes = false;
+    }
+
+    public function testCreateMediaWithSetPreviewDrivers()
+    {
+        $this->mockGenerateFilename();
+
+        $media = app(MediaServiceContract::class)
```

Comment:
Let's use multi line syntax for create method. please implement this in other methods UnitTest  class.
```suggestion
        $media = app(MediaServiceContract::class)->create(
           arg1: $arg1,
           arg2: $arg2,
        );
```

Reply:
not accepted